src/App.js

```diff
@@ -31,12 +36,24 @@ export default function App() {
           </Route>
         ))}
       </Switch>
-      <Navigation />
+      {play ? (
+        <BottomContainer>
+          <AudioPlayer audioObject={play} />
+          <Navigation />
+        </BottomContainer>
+      ) : (
+        <Navigation />
+      )}
     </AppContainer>
   )
 }
 
 const AppContainer = styled(Div100vh)`
   display: grid;
-  grid-template-rows: 3rem 1fr 3rem;
+  grid-template-rows: 3rem 1fr ${props => (props.isPlay ? '6.1rem' : '3rem')};
```

Comment:
interesting value of 6.1rem 🙃

Reply:
thank you for review :) Yes thats accounting for the height of both navigation as well as audioplayer and the gap between those.